apiary.apib

```diff
@@ -1943,6 +1943,22 @@ Get details for a single invoice.
             + created_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
 
+### invoices.download [POST /invoices.download]
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
+        + format: `pdf` (enum[string], required)
+            + Members
+                + pdf
+
++ Response 307 (application/json;charset=utf-8)
+
+    + Headers
+
+              Location: The url of the downloadable file. This will remain valid for 30 seconds.
```

Comment:
Q: the indentation looks quite weird compared to the other endpoints. Is this intended?

Reply:
https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#example-18

Reply:
TIL

 :ok: by @WouterSioen on 30/05/2018 10:50:55